### ISSUE TITLE
Write fails, not enough free space - Macos

### ISSUE THREAD

**Issue by WGandy** [s3fs-fuse/s3fs-fuse#1982]
Perhaps I'm missing something simple here but here goes. 

#### Version of s3fs being used (s3fs --version)
Amazon Simple Storage Service File System V1.91

#### Version of fuse being used (pkg-config --modversion fuse, rpm -qi fuse, dpkg -s fuse)
$ pkg-config --modversion fuse                            
2.9.7

#### GNU/Linux Distribution, if applicable (cat /etc/os-release)
Macos Big Sur 11.6.7

### Details about issue
In finder the status bar shows 1MB of free space when I'm browsing the S3 bucket. If I try to copy a file (50MB or so) into the folder it spends some time preparing and then I get this error 
`The item “myfilename” can’t be copied because there isn’t enough free space.`

It seems that there needs to be some kind of cache for writes on the Mac machine? The bucket has plenty of space for this.

**Comment by suminb** [comment]
I have a similar issue. `df -h` command shows the following (note that the volume size is 1MB):

```
Filesystem       Size   Used  Avail Capacity iused       ifree %iused  Mounted on
...
s3fs@macfuse3   1.0Mi    0Bi  1.0Mi     0%       0           0  100%   /some/mount/point
```

Is there any way to manually adjust the volume size?

**Comment by ggtakec** [comment]
@WGandy @suminb I'm sorry for late reply.
I uploaded the fix in #2037.
I think the corrections for sizes you can see by df command are fixed in this PR.
If you can, please build the code in that PR and give it a try.

The PR code will be reviewed and then merged into master.

**Comment by ggtakec** [comment]
PR #2037 was merged.
If you can try it with master's code, please try it.
We will close this issue, but if the problem still exists, please reopen it.
